tests/fixtures/references.eml

```diff
@@ -0,0 +1,10 @@
+Message-ID: <123@example.com>
+From: no_host
+Cc: "This one: is \"right\"" <ding@dong.com>, No-address
+References: <231d9ac57aec7d8c1a0eacfeab8af6f3@example.com> <08F04024-A5B3-4FDE-BF2C-6710DE97D8D9@example.com>

```

Comment:
Would you be so kind to also add the same API for `In-Reply-To` header?
https://tools.ietf.org/html/rfc4021#section-2.1.9
https://tools.ietf.org/html/rfc2822#section-3.6.4
It seems to have exactly the same behaviour.

Reply:
Check this 2b4f5af